tests/unit/test-class-wc-payments-features.php

```diff
@@ -72,33 +73,18 @@ function ( $pre_option, $option, $default ) {
 	}
 
 	public function test_is_platform_checkout_is_returned_as_true() {
```

Comment:
This `test_is_platform_checkout_is_returned_as_true` kinda looks misspelled. Maybe it was meant to be `test_if`... or even `test_is_platform_checkout_eligible`